src/code/z_sound_source.c

```diff
@@ -9,24 +9,24 @@ void func_8006BA00(GlobalContext* globalCtx) {
     // clang-format on
 }
 
-void func_8006BA30(GlobalContext* globalCtx) {
+void SoundSource_UpdateAll(GlobalContext* globalCtx) {
     SoundSource* source = &globalCtx->soundSources[0];
     s32 i;
 
     for (i = 0; i < ARRAY_COUNT(globalCtx->soundSources); i++) {
         if (source->countdown != 0) {
             if (DECR(source->countdown) == 0) {
-                Audio_StopSfxByPos(&source->relativePos);
+                Audio_StopSfxByPos(&source->projectedPos);
             } else {
-                SkinMatrix_Vec3fMtxFMultXYZ(&globalCtx->viewProjectionMtxF, &source->originPos, &source->relativePos);
+                SkinMatrix_Vec3fMtxFMultXYZ(&globalCtx->viewProjectionMtxF, &source->originWorldPos, &source->projectedPos);
             }
         }
 
         source++;
     }
 }
 
-void Audio_PlaySoundAtPosition(GlobalContext* globalCtx, Vec3f* pos, s32 duration, u16 sfxId) {
+void SoundSource_PlaySfxAtStationaryPosition(GlobalContext* globalCtx, Vec3f* worldPos, s32 duration, u16 sfxId) {
```

Comment:
I think my stance on this was along the lines of
```suggestion
void SoundSource_PlaySfxAtFixedWorldPos(GlobalContext* globalCtx, Vec3f* worldPos, s32 duration, u16 sfxId) {
```
and there was no other issue for me

Reply:
Yeah, I guess I can concede. Now that I know how the system fully works, I think that name is fine

Reply:
Alright, this name has been changed. And with this new name, I also agree more with changing the SoundSource struct's `originWorldPos` to just `worldPos`, so that's been changed too.